OMNA/security/gba_protocol_identifiers.html

```diff
@@ -0,0 +1,157 @@
+<!doctype html>
+<html>
+<head>
+
+    <title>Open Mobile Alliance - GBA Protocol Identifiers</title>
+    <meta charset="utf-8">
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"></script>
+    <script src="/Scripts/LwM2MRegistry.js"></script>
+
+    <!-- Font awesome for arrows -->
+    <link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css">
+
+    <!-- Normalize CSS -->
+    <link rel="stylesheet" href="/CSS/Normalize.css" />
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap-theme.min.css">
+
+    <link rel="stylesheet" href="/CSS/Site.css">
+    <link rel="stylesheet" href="/CSS/LwM2MRegistry.css">
+
+    <script>
+        $(function () {
+            $('#header').load("/header.html");
+            $('#footer').load("/footer.html");
+            $('#LwM2MObjects').load("LwM2MObjects.html");
```

Comment:
I agree with this; it is not needed.

The header was copied from openmobilealliance.github.io/OMNA/LwM2M/LwM2MRegistry.html . Are there any further changes needed to the registries using this template e.g. perhaps the references to LwM2MRegistry.js and LwM2MRegistry.css? It would be good to be able to make all the changes necessary in one go.